src/gtk/choice.cpp

```diff
@@ -111,6 +111,20 @@ wxChoice::~wxChoice()
  #endif // __WXGTK3__
 }
 
+bool wxChoice::GTKHandleFocusOut()
+{
+    gboolean isShown;
+    g_object_get( m_widget, "popup-shown", &isShown, NULL );
```

Comment:
`"popup-shown"` was introduced in 2.10, so this will need a version check to avoid `GLib-GObject-WARNING **: g_object_get_is_valid_property: object class 'GtkComboBox' has no property named 'popup-shown'`

Reply:
Thanks, added. Just out of curiosity, do you still have a pre-2.10 version of GTK+ you're testing with?